gbi.h

```diff
@@ -1145,14 +1146,14 @@ typedef struct {
 } Light_t;
 
 typedef struct {
-    unsigned char col[3];   /* point light color (rgb) */
-    unsigned char kc;       /* point light enable flag (> 0) & constant attenuation Kc */
-    unsigned char colc[3];  /* copy of point light color (rgb) */
-    unsigned char kl;       /* linear attenuation Kl */
-    short pos[3];           /* light position x, y, z in world space */
-    unsigned char kq;       /* quadratic attenuation Kq */
+    unsigned char col[3];   /* point light value (rgba) */
```

Comment:
The comment change for this line and `colc` is not an improvement, it's the point light color and it is RGB only, it is not RGBA and I think "light value" is less clear than light color. If this is from upstream I would suggest fixing it there too.

Reply:
sgi's gbi.h uses "value (rgba)" in the `Light_t` structure which I synced the comments with. I've fixed it here to match the new more correct wording in the f3dex3 gbi.h, but not sure whether I'll change the wording upstream yet.